thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -59,25 +60,24 @@ export default Controller.extend({
       isReportFailure: false,
       isPageLoadFailure: false,
       isReplayModeWrapper: true,
+      isReplayStatusError: false,
       isAnomalyArrayChanged: false,
       requestCanContinue: true,
       sortColumnStartUp: false,
       sortColumnScoreUp: false,
       sortColumnChangeUp: false,
       sortColumnResolutionUp: false,
-      checkReplayInterval: 120000, // 2min
+      checkReplayInterval: 2000, // 2 seconds
```

Comment:
Just curious, is this an arbitrary number?

Reply:
yes